src/molecule/provisioner/ansible.py

```diff
@@ -940,7 +940,7 @@ def _get_modules_directories(self):
         )
 
         if os.environ.get("ANSIBLE_LIBRARY"):
-            paths.extend(map(util.abs_path, os.environ["ANSIBLE_LIBRARY"].split(":")))
+            paths.insert(0, map(util.abs_path, os.environ["ANSIBLE_LIBRARY"].split(":")))
```

Comment:
I think it's better to move this up to L918 with something like...

        paths = []
        if os.environ.get("ANSIBLE_LIBRARY"):
            paths.extend(map(util.abs_path, os.environ["ANSIBLE_LIBRARY"].split(":")))
        paths.extend([util.abs_path(os.path.join(self._get_plugin_directory(), "modules"))])

Reply:
done.